robottelo/constants/__init__.py

```diff
@@ -310,7 +310,13 @@
 
 SYNC_INTERVAL = {'hour': "hourly", 'day': "daily", 'week': "weekly", 'custom': "custom cron"}
 
-REPO_TYPE = {'yum': "yum", 'puppet': "puppet", 'docker': "docker", 'ostree': "ostree"}
+REPO_TYPE = {
+    'yum': "yum",
+    'puppet': "puppet",
```

Comment:
Note: puppet repo type is no longer supported in 6.10, but I guess this will be part of other PR